anvio/docs/workflows/ecophylo.md

```diff
@@ -0,0 +1,76 @@
+The %(ecophylo-workflow)s explores the ECOlogical and PHYlogenetic relationships of proteins across genomes and metagenomes. It is a [Snakemake](https://snakemake.readthedocs.io/en/stable/) workflow run by the anvi'o script %(anvi-run-workflow)s. Briefly, the workflow extracts a target protein from any number of assemblies in %(metagenomes)s and/or %(external-genomes)s using a user-designated [HMM](https://anvio.org/vocabulary/#hidden-markov-models-hmms) from %(hmm-list)s, then clusters the sequences and selects representatives from each cluster. Next, the workflow post-processes the representative sequences using phylogenetics and metagenomic read recruitment to produce an anvi'o %(interactive)s interface to explore their phylogenetic distances and co-occurrence across metagenomes (via simultaneous visual of a phylogenetic tree and read recruitment results). The workflow can use any protein-based [HMM](https://anvio.org/vocabulary/#hidden-markov-models-hmms) including [single-copy core genes](https://anvio.org/vocabulary/#single-copy-core-gene-scg) to taxonomically profile metagenomes or any functional protein to explore variants across samples. 
```

Comment:
We will get rid of all `%(*-workflow)s` artifacts. You should simply call it clear-text "The ecophylo workflow explores (...)"

Statements like these are not necessary either:

```
It is a [Snakemake](https://snakemake.readthedocs.io/en/stable/) workflow run by the anvi'o script %(anvi-run-workflow)s
```

Because they are generic information for anvi'o workflows.

Reply:
If you like, at this point in the text you could also put a simple figure to show people an example output (perhaps annotated in such a way they know these are the metagenomes, that is the phylogenetic tree, etc) :)

Reply:
Good call, I don't have a solid example yet but I will add one in the future.